neqo-http3/src/recv_message.rs

```diff
@@ -304,6 +321,23 @@ impl RecvMessage {
             RecvMessageState::ClosePending | RecvMessageState::Closed
         )
     }
+
+    fn is_interim(&self, headers: &[Header]) -> Res<bool> {
+        match self.message_type {
+            MessageType::Response => {
+                let status = headers.iter().find(|e| e.0 == ":status");
```

Comment:
This is probably is a case where find_map doesn't make the code more readable, sadly.

```suggestion
                let status = headers.iter().find(|(name, _value)| name == ":status");
```